### PR TITLE
refactor: change var instances to const or let

### LINKED ISSUE #5756: Use of `var` for initialising variables
Noticed this whilst looking at https://github.com/fastify/fastify/pull/5745, Fastify has a few `var`s knocking about in both [`fastify` itself](https://github.com/search?q=repo%3Afastify%2Ffastify%20var&type=code) and supporting plugins, primarily in loops.

Is there a performance reason for using `var` over `let` or `const`?

### PR DESCRIPTION
**Pull request by ChinoUkaegbu**
Closes #5756

#### Checklist

- [X] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by ChinoUkaegbu** [comment]
The [configValidator](https://github.com/fastify/fastify/blob/e61c394745259f79bc96afbb80e38dd512bcd437/lib/configValidator.js) file also has instances of `var` but the comment at the top indicates that it's not to be edited.

File: lib/contentTypeParser.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ ContentTypeParser.prototype.getParser = function (contentType) {
   }
 
   // eslint-disable-next-line no-var
```
Comment (ChinoUkaegbu) [inline lib/contentTypeParser.js:133]: is this to be left in?

File: lib/contentTypeParser.js [hunk 2]
```diff
@@ -114,7 +114,7 @@ ContentTypeParser.prototype.getParser = function (contentType) {
 
   const caseInsensitiveContentType = contentType.toLowerCase()
   // eslint-disable-next-line no-var
```
Comment (gurgunday) [inline lib/contentTypeParser.js:116]: You need to remove these

Comment (ChinoUkaegbu) [reply lib/contentTypeParser.js]: done!

**Review by dancastillo — APPROVED** [review]

**Review by jsumners — APPROVED** [review]

**Review by gurgunday — APPROVED** [review]

**Pull request merged**